Bluetooth/doc/BluetoothPlugin.md

```diff
@@ -1192,6 +1303,52 @@ Unpairs the given device with this device.
 }
 ```
 
+<a name="method.getApiVersionNumber"></a>
+## *getApiVersionNumber <sup>method</sup>*
+
+Provides the current API version number. 
+ 
+### Event 
+
+ No Events.
+
+### Parameters
+
+This method takes no parameters.
+
+### Result
+
+| Name | Type | Description |
+| :-------- | :-------- | :-------- |
+| result | object |  |
+| result.version | integer | API Version Number |
+| result.success | boolean | Whether the request succeeded |
+
+### Example
+
+#### Request
+
+```json
+{
+    "jsonrpc": "2.0",
+    "id": 1234567890,
+    "method": "org.rdk.Bluetooth.1.getApiVersionNumber"
+}
+```
+
+#### Response
+
+```json
+{
+    "jsonrpc": "2.0",
+    "id": 1234567890,
+    "result": {
+        "version": 1,
+        "success": true
+    }
+}
+```
+
 <a name="head.Notifications"></a>
 # Notifications
```

Comment:
I think these two are missing in  Notifications
**deviceFound**	
HashMap {


deviceID:uint64

name:String

deviceType:String

rawDeviceType:number

lastConnectedState:bool

}
	

One or more of third party devices, that have already been paired to the set top box, will send this event when they come inside the set top box range or gets powered Up within the range.

Separate notification is sent for each device that comes up, provided the device is already paired with the set top box.

deviceID - ID derived from Bluetooth MAC address. 6 byte MAC value is packed into 8 byte with leading zeros for first 2 bytes.

name - device name as specified by manufacturer

deviceType - device class

rawDeviceType : Bluetooth device class hex code

lastConnectedState - true or false. Only the last connected device has the true status.

**deviceLost**	
HashMap {


deviceID:uint64

name:String

deviceType:String

rawDeviceType:number

lastConnectedState:bool

}

Reply:
Now, "onDeviceFound" and "onDeviceLost" added in Notifications.